### PR TITLE
Add org.zulip.Zulip

### PR DESCRIPTION
**Pull request by ghisvail**
Once built can be tested with the Zulip project's chat instance hosted at chat.zulip.org.

### CODE REVIEW

**Comment by TingPing** [comment]
bot, build org.zulip.Zulip

**Comment by nedrichards** [comment]
Needs version information and OARS in the appdata. It'd also be great to see a PR sending the appdata upstream for a future release.

**Comment by ghisvail** [comment]
> It'd also be great to see a PR sending the appdata upstream for a future release.

I have proposed it in their support channel at chat.zulip.org, and we agreed to wait for the completion of this review so I can submit the final version of the file.

**Comment by nedrichards** [comment]
bot, build org.zulip.Zulip

File: org.zulip.Zulip.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        "ar -x Zulip-2.0.0-amd64.deb",
+        "tar -xf data.tar.xz",
+        "cp -r usr/* /app",
+        "mkdir -p /app/bin",
+        "cp -r opt/Zulip/* /app/bin",
+        "desktop-file-edit --set-key=Exec --set-value='zulip %U' /app/share/applications/zulip.desktop",
+        "install -Dm644 org.zulip.Zulip.appdata.xml /app/share/appdata/org.zulip.Zulip.appdata.xml"
+      ],
+      "sources": [
+        {
```
Comment (barthalion) [inline org.zulip.Zulip.json]: Please add source tarball as well so it's included into Sources extension.

Comment (ghisvail) [reply org.zulip.Zulip.json]: Done

**Comment by nedrichards** [comment]
bot, build org.zulip.Zulip

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.zulip.Zulip

**Pull request closed without merging**